spec/integration/soap_connection_spec.rb

```diff
@@ -0,0 +1,33 @@
+require 'spec_helper'
+require 'nokogiri'
+
+describe Zuora::SoapClient do
+  let(:username) { ENV['ZUORA_SANDBOX_USERNAME'] }
+  let(:password) { ENV['ZUORA_SANDBOX_PASSWORD'] }
+  let(:client) { Zuora::SoapClient.new(username, password, true) }
+  let!(:auth_response) do
+    VCR.use_cassette('soap_authentication', match_requests_on: [:path]) do
+      client.authenticate!
+    end
+  end
+  let!(:create_bill_run_response) do
+    VCR.use_cassette('soap_create_bill_run', match_requests_on: [:path]) do
```

Comment:
Can you add specs for a failed bill run?  Similar to the other specs with the same cassette naming pattern, e.g.

```
VCR.use_cassette('soap_create_bill_run_success', match_requests_on: [:path])
```